docs/logging/log-schema.md

```
@@ -0,0 +1,118 @@
+# SSD 운영 로그 스키마
+
+## 1. 기본 원칙
+
+| 원칙 | 내용 |
+| --- | --- |
+| 추적 기준 | 모든 애플리케이션 로그는 `requestId`를 기준으로 연결합니다. |
+| 라벨 기준 | Loki label은 `env`, `service`, `logType`, `level`처럼 cardinality가 낮은 값만 사용합니다. |
+| 검색 기준 | `requestId`, `memberId`, `documentId`, `folderId`는 JSON field로 남기고 LogQL `| json` 이후 검색합니다. |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)